doc/usermanual/darkroom/modules/tone/filmic.xml

```diff
@@ -0,0 +1,249 @@
+          almost centered in 0 EV. Another guide to validate the black exposure value is the S curve should always go
+          through the center of the graph display.
+        </para>
+        
+      </sect6>
+      
+      <sect6>
+      
+        <title>Safety factor and auto-tune</title>
+        
+        <para>
+          The auto-tune color picker combines all three color-pickers above, and will allow to set the grey, white 
+          and black exposures all at once, using the average of the drawn region as the grey estimation, the maximum as the white, 
+          and the minimum as the black. This gives good results in landscape photography but usually fails for portraits
+          and indoors scenes.
+        </para>
+        <para>
+          When no true white and no true black are available on the scene, the maximum and minimum XYZ values read on the
+          image are not valid assumptions anymore, so the safety-factor will allow you to shrink or enlarge symmetrically
+          the detected dynamic range and the current parameters. This works will all color-pickers.
+        </para>
+        
+      </sect6>
+      
+    </sect5>
+
+    <sect5>
+      <title>Filmic S curve</title>
+      <para>
+        The filmic S curve is created from the user parameters, by computing the position of virtual nodes and interpolating them,
+        similarly to the tone curve module (but here, the nodes cannot be moved manually). While this allow a quicker curve
+        parametrization, not all the combinations of paramater deliver good curves, and the graph display will give a 
+        diagnostic interface. Good curves are either graceful S with a linear central portion and curved extremities,
+        or lines following the diagonal of the graph. 
+      </para>
+      
+      <sect6>
+      
+        <title>Contrast</title>
+        
+        <para>
+          The contrast parameter drives the slope of the central part of the curve. The larger the dynamic range is, the greater 
+          the contrast should be set, to avoid reversed branches that would invert the image contrast. Proper values of contrast
+          lie between 1.4 (at 8 EV dynamic range) and 1.7 (at 14 EV dynamic range). This parameter will mostly affect
+          mid-tones.
+        </para>
+        
+      </sect6>
+      
+      <sect6>
+      
+        <title>Latitude</title>
+        
+        <para>
+          The latitude is the range between the 2 nodes enclosing the central linear portion of the curve. The bigger the latitude is,
+          the more linearly the luminance is affected, which is desirable to preserve the chrominance. The latitude defines
+          the range of luminances that is not desaturated in the third step of filmic processing. This parameter will mostly
+          affect extreme luminances. It is usually better to keep it as great as possible, and adjust the contrast to avoid clipping. 
```

Comment:
```suggestion
          affect extreme luminances. It is usually better to keep it as large as possible, and adjust the contrast to avoid clipping. 
```

?

Reply:
done